packages/web/components/trade-clipboard/index.tsx

```diff
@@ -388,14 +388,14 @@ export const TradeClipboard: FunctionComponent<{
     return (
       <div
         className={classNames(
-          "relative rounded-[18px] flex flex-col gap-8 md:gap-6 bg-osmoverse-800 px-6 md:px-3 pt-12 md:pt-4 pb-8 md:pb-4",
```

Comment:
Can we clip the bottom of the drawer except on mobile? Would look nice to contain the drawer within the swap widget.

Reply:
I wanted to do this, but the height of the swap widget is too small so not much content fits the select. I thought about increasing the `min-height` but that would change the overall design. I'll leave a comment in design to see what we can do

Reply:
Sounds fair. I sent a message to @syed8rahman to see what he thinks about the popup not being clipped. I think it's a bit odd for it not to clip _and_ for there to be a negative area at the top of the tool that can be used to dismiss it.